percy/common/common.rb

```diff
@@ -2,18 +2,21 @@
 
 require 'dotenv/load'
 
-PERCY_LOGLEVEL = ENV['PERCY_LOGLEVEL']
-PERCY_DEBUG = PERCY_LOGLEVEL == 'debug'
-LABEL = "[\e[35m#{PERCY_DEBUG ? 'percy:ruby' : 'percy'}\e[39m]"
 
 def log(message, on_debug: nil)
-  return unless on_debug.nil? || (on_debug.is_a?(TrueClass) && PERCY_DEBUG)
+  return unless on_debug.nil? || (on_debug.is_a?(TrueClass) && percy_debug)
```

Comment:
changed this because PERCY_DEBUG is a constant and we can't change the value of it mid session.
Also it is causing issue in writing test cases as updating env variable doesn't reflect in changing PERCY_DEBUG value.
 new function percy_debug seems to be a better option.